### DISCUSSION TITLE
Auto generate getters and setters

### DISCUSSION THREAD

**Discussion by sovetski** [nette/php-generator#161] [Ideas]
Hi,

I know that using getters and setters depends on the preferences of developers, but it is the default approach in frameworks like Symfony.

I didn't find anything related to this in the library. Do we have the option to "generate getters and setters from properties"?

If not, is it possible to add this feature? If the maintainers are okay with it, I can look into creating a PR.

Thanks.

**Comment by dg** [comment]
You can send a PR. Ideally try to add functionality to the https://github.com/nette/php-generator/blob/master/src/PhpGenerator/ClassManipulator.php
